interface-definitions/load-balancing_reverse-proxy.xml.in

```diff
@@ -102,6 +102,45 @@
                   </leafNode>
                 </children>
               </node>
+              <node name="http-check">
+                <properties>
+                  <help>HTTP check configuration</help>
+                  <completionHelp>
+                    <list>uri /health</list>
+                  </completionHelp>
```

Comment:
It is not required and looks strange, as you use the next leafnodes `method` and `uri`

Reply:
removed